CommandLine.c

```diff
@@ -411,6 +411,8 @@ int CommandLine_run(int argc, char** argv) {
    CRT_done();
 
    if (settings->changed) {
+      if (!String_eq(settings->initialFilename, settings->filename))
```

Comment:
I would put this into #ifndef NDEBUG as it is too noisy for affected users in release builds.

Reply:
```diff --git a/CommandLine.c b/CommandLine.c
index e4564564..69b80c1e 100644
--- a/CommandLine.c
+++ b/CommandLine.c
@@ -411,8 +411,10 @@ int CommandLine_run(int argc, char** argv) {
    CRT_done();
 
    if (settings->changed) {
+#ifndef NDEBUG
       if (!String_eq(settings->initialFilename, settings->filename))
          fprintf(stderr, "Configuration %s was resolved to %s\n", settings->initialFilename, settings->filename);
+#endif /* NDEBUG */
       int r = Settings_write(settings, false);
       if (r < 0)
          fprintf(stderr, "Can not save configuration to %s: %s\n", settings->filename, strerror(-r));

```

Reply:
I am totally on board with this. I added this as it was suggested by @Explorer09